provider-directory/src/main/java/gov/va/api/health/providerdirectory/service/ProviderServicesResponse.java

```diff
@@ -0,0 +1,103 @@
+package gov.va.api.health.providerdirectory.service;
+
+import com.fasterxml.jackson.annotation.JsonAutoDetect;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import java.util.ArrayList;
+import java.util.List;
+import lombok.AccessLevel;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@Builder
+@JsonIgnoreProperties(ignoreUnknown = true)
+@NoArgsConstructor(access = AccessLevel.PRIVATE)
+@AllArgsConstructor(access = AccessLevel.PRIVATE)
+@JsonAutoDetect(fieldVisibility = JsonAutoDetect.Visibility.ANY)
+public final class ProviderServicesResponse implements PpmsResponse {
+  @JsonProperty("@odata.context")
+  private String odataContext;
+
+  private List<Value> value;
+
+  private Error error;
+
+  /** Lazy getter. */
+  public List<Value> value() {
+    if (value == null) {
+      value = new ArrayList<>();
+    }
+    return value;
+  }
+
+  @Data
+  @Builder
+  @JsonIgnoreProperties(ignoreUnknown = true)
+  @NoArgsConstructor(access = AccessLevel.PRIVATE)
+  @AllArgsConstructor(access = AccessLevel.PRIVATE)
+  @JsonAutoDetect(fieldVisibility = JsonAutoDetect.Visibility.ANY)
+  public static final class Value {
+    @JsonProperty("Name")
+    private String name;
+
+    @JsonProperty("AffiliationName")
+    private String affiliationName;
+
+    @JsonProperty("RelationshipName")
+    private String relationshipName;
+
+    @JsonProperty("ProviderName")
+    private String providerName;
+
+    @JsonProperty("ProviderAgreementName")
+    private String providerAgreementName;
+
+    @JsonProperty("SpecialtyName")
+    private String specialtyName;
+
+    @JsonProperty("HPP")
+    private String hpp;
+
+    @JsonProperty("HighPerformingProvider")
+    private String highPerformingProvider;
+
+    @JsonProperty("CareSiteName")
+    private String careSiteName;
+
+    @JsonProperty("CareSiteAddressStreet")
+    private String careSiteAddressStreet;
+
+    @JsonProperty("CareSiteLocationAddress")
+    private String careSiteLocationAddress;
+
+    @JsonProperty("CareSiteAddressCity")
+    private String careSiteAddressCity;
+
+    @JsonProperty("CareSiteAddressState")
+    private String careSiteAddressState;
+
+    @JsonProperty("CareSiteAddressZipCode")
+    private String careSiteAddressZipCode;
+
+    @JsonProperty("Latitude")
+    private String latitude;
+
+    @JsonProperty("Longitude")
+    private String longitude;
+
+    @JsonProperty("CareSitePhoneNumber")
+    private String careSitePhoneNumber;
+
+    @JsonProperty("OrganiztionGroupName")
+    private String organiztionGroupName;
```

Comment:
Typo?

Reply:
Is `OrganiztionGroupName` how it's spelled in PPMS?

Reply:
It comes back as a typo from PPMS, I will change our naming of it to correct the typo on our side, but the JsonProperty will still have the 'typo' to ensure mapping works.